### ISSUE TITLE
Better CI Build tags

### ISSUE THREAD

**Issue by crutkas** [Microsoft/WindowsTemplateStudio#843]
**Overview:** I saw what UWP community toolkit is doing and I will admit, I like it.  It shows what tests, artifacts and what built.
**Links:**  

- https://ci.appveyor.com/project/dotnetfoundation/uwpcommunitytoolkit/branch/dev
- https://github.com/Microsoft/UWPCommunityToolkit

**their build status:**
![image](https://user-images.githubusercontent.com/1462282/28433376-2e2f7578-6d40-11e7-837a-5888cc225458.png)

**our build status:**
![image](https://user-images.githubusercontent.com/1462282/28433403-445521c2-6d40-11e7-9c4f-9662282b89f7.png)

**Comment by mrlacey** [comment]
As per comments in (linked) #980 a build failure should not be silent and should send [email] notifications.
Or it could raise a new issue. 
Raising a new issue would be better (IMO) as this wouldn't be hidden or lost in someone's email inbox and multiple people could get the email if they wish (by subscribing to email notifications for new issues.)

**Comment by crutkas** [comment]
@mrlacey great idea.

**Comment by ralarcon** [comment]
Let me check how to create the issue once a build fails (apart from email the build result, which I can configure...)

Regarding what UWP Toolkit guys have, we can move some of our CI builds to AppVeyor to enable people see the build result, but others may not be possible as we use some tasks to upload elements our azure storage accounts...

**Comment by crutkas** [comment]
hmmm, valid point.  I would view this as low priority honestly but is a nice to have

**Comment by ralarcon** [comment]
First step. I have created a CI in AppVeyor for at least enable developers check what is going to happen with their PRs / pushes as well as help reviewers who can verify that the PR is able to merge and build.

Trade offs: 
* It is not possible to build the extension project in AppVeyor (at least at a first glance), so the CI is not going to build the extension.
* By now, the tests which are being executed at the same time as this CI are the Core.Tests

**Comment by ralarcon** [comment]
We are not going to be able to move all builds to AppVeyor (at least in a easy / stright forward way). I have include two builds to help developers check what is going to happen with their PRs.

I'm going to close this now. @crtukcas @mrlacey Feel free to reopen / create a new Issue if you think it is needed to invest more in this space.

**Comment by crutkas** [comment]
awesome.  so front page won't have updates but at least the merges will be blocked on failures?
